ci/keep_alive

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Seems like bazel sometimes will not print progress output even with the interval set, so use a script to make sure it happens.